### PR TITLE
Focus iOS export Apr 7 2022

### PR DESCRIPTION
**Pull request by nbhasin2**

### CODE REVIEW

File: templates/focus-ios.xliff [hunk 1]
```diff
@@ -381,6 +381,42 @@
...
+      <trans-unit id="Onboarding.History.Description">
+        <source>Erase your browsing history, passwords, cookies, and prevent unwanted ads from following you in a simple click!</source>
+        <note>Text for a label that indicates the description of history section from onboarding screen.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.History.Title">
+        <source>Your history doesn’t follow you</source>
+        <note>Text for a label that indicates the title of history section from onboarding screen.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.Incognito.Description">
+        <source>Focus is a dedicated privacy browser with tracking protection and content blocking.</source>
```
Comment (Delphine) [inline templates/focus-ios.xliff:397]: "Focus" should be a placeholder since it also can be "Klar"

File: templates/focus-ios.xliff [hunk 2]
```diff
@@ -381,6 +381,42 @@
...
+        <source>Protection at your own discretion</source>
+        <note>Text for a label that indicates the title of protection section from onboarding screen.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.Subtitle">
+        <source>Take your private browsing to the next level.</source>
+        <note>Text for a label that indicates the subtitle for onboarding screen.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.Title">
+        <source>Welcome to Firefox %@!</source>
+        <note>Text for a label that indicates the title for onboarding screen. (Focus and Klar)</note>
```
Comment (Delphine) [inline templates/focus-ios.xliff:418]: Please comment explicitely that the placeholder will be replaced by app name (eg Focus or Klar)

File: templates/focus-ios.xliff [hunk 3]
```diff
@@ -829,6 +865,26 @@
...
+      <trans-unit id="TooltipBodyText.ContextMenu">
+        <source>Go to Settings to manage specific privacy &amp; security options.</source>
+        <note>This is the body text that is displayed for the Context Menu icon tooltip</note>
+      </trans-unit>
+      <trans-unit id="TooltipBodyText.Privacy">
+        <source>These default settings offer strong protection. But it’s easy to tweak the settings to meet your specific needs.</source>
+        <note>This is the body text that is displayed for the Privacy tooltip</note>
+      </trans-unit>
+      <trans-unit id="TooltipBodyText.ShieldIcon">
+        <source>Focus stopped this site from spying on you. Tap the shield for info on what we blocked.</source>
```
Comment (Delphine) [inline templates/focus-ios.xliff:877]: Placeholder for app name

**Review by Delphine — CHANGES_REQUESTED** [review]
@nbhasin2 Thanks for the export! just a few things to correct

File: templates/focus-ios.xliff (hunk 1 above)
Comment (nbhasin2) [reply templates/focus-ios.xliff:397]: corrected here: https://github.com/mozilla-l10n/focusios-l10n/pull/80/files#diff-5aaa6e083e6c89a11d87391ecf7ab52b669793c5ffc9c713b2471b628fb04959R462-R463

File: templates/focus-ios.xliff (hunk 2 above)
Comment (nbhasin2) [reply templates/focus-ios.xliff:418]: corrected here: https://github.com/mozilla-l10n/focusios-l10n/pull/80/files#diff-32781eb8e0a5f13c0370cf3bf8653e7bb3a02d5269b57edfc8266c10d272c7b3R505-R508

File: templates/focus-ios.xliff (hunk 3 above)
Comment (nbhasin2) [reply templates/focus-ios.xliff:877]: corrected here: https://github.com/mozilla-l10n/focusios-l10n/pull/80/files#diff-32781eb8e0a5f13c0370cf3bf8653e7bb3a02d5269b57edfc8266c10d272c7b3R1050-R1052

**Comment by nbhasin2** [comment]
Closing this in favor of https://github.com/mozilla-l10n/focusios-l10n/pull/80

**Pull request closed without merging**